bbc/simorgh
# Change News OJ title from "Features & Analysis" to "More to explore"

Resolves JIRA [number]

Overall changes
======
* Changes heading on News 'Features' onward journeys from "Features & Analysis" to "More to explore"

Code changes
======

- Adds new translation to News service config file

Testing
======
1. Visit a news .amp page
2. Observe the RHS onward journeys
3. See title on the second onward journey component
4. Verify it reads "More to explore"

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
